.github/runlint.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+###############################################################################
+# Copyright (c) 2021, 2021 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+# Exit immediately if any unexpected error occurs.
+set -e
+# trace the commands that are run
+set -x
+
+MAKE_JOBS=4
+
+# explicitly select the desired version of GCC via the environment
+export CC=gcc-7
+export CXX=g++-7
+
+# download bootstrap jdk
+SRCDIR=$PWD
+cd ~
+wget -O bootjdk11.tar.gz https://api.adoptopenjdk.net/v3/binary/latest/11/ga/linux/x64/jdk/openj9/normal/adoptopenjdk
+tar -xzf bootjdk11.tar.gz
+rm -f bootjdk11.tar.gz
+mv $(ls | grep -i jdk) bootjdk11
+export JAVA_HOME="$PWD/bootjdk11"
+export PATH="${JAVA_HOME}/bin:${PATH}"
+cd $SRCDIR
+
```

Comment:
`build-on-travis.sh` included these two commands that I think are still worthwhile:
```
llvm-config --version
clang++ --version
```

Reply:
Not sure what's the purpose of those two commands? I can see under the travis environment the output is 7.0 and we actually are using 3.8? 

```
+llvm-config --version
7.0.0
+clang++ --version
clang version 7.0.0 (tags/RELEASE_700/final)
```
Looks like clang and llvm 7 are just default travis-ci Compilers and Build toolchain so I guess the two commands were just to verify travis-ci environment?

**Default Compilers and Build toolchain #
clang and llvm 7
cmake 3.12.4
gcc 5.4.0
ccache 3.2.4
shellcheck 0.7.0
shfmt 2.6.3**

https://docs.travis-ci.com/user/reference/xenial/

git-hub runner environment is different from travis-ci. No default llvm and clang is Clang 6.0.0, 8.0.0, 9.0.1.
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1604-README.md

That means means if we install dependencies by **sudo apt-get -y install clang-3.8 llvm-3.8** and run **llvm-config --version** we will get **llvm-config: command not found**. Unless extra link command to link llvm-config to 3.8

Reply:
I support the commands would need to be:
```
$LLVM_CONFIG --version
$CXX --version
```
But then, it would only server to confirm what we're getting.